### DISCUSSION TITLE
Where to get the vocab file for Chinese?

### DISCUSSION THREAD

**Discussion by HenryYuen128** [openai/whisper#912] [Q&A] [answered]
I was fine-tuning the model with Chinese data, where can i get the vocab file for chinese?

**Comment by tnlin** [comment]
It uses byte-level BPE; I'm still trying to understand how the multilingual tokenizer works with the given vocab

**Comment by jongwook** [comment, marked as answer]
The tokenizer is byte-pair encoding (BPE) using UTF-8 bytes, so it can encode arbitrary unicode strings. We don't have an encoding specific to Chinese, but the BPE vocabs used for the multilingual Whisper models were trained on the entire training dataset, so it will encode Chinese text quite efficiently.
